src/plugins/explore/public/components/visualizations/visualization_builder.ts

```diff
@@ -277,9 +275,15 @@ export class VisualizationBuilder {
           this.setStyles({ styles: visConfig?.ui.style.defaults, type: autoVis.chartType });
         }
       } else {
-        this.setCurrentChartType(undefined);
+        const visConfig = visualizationRegistry.getVisualizationConfig('table');
```

Comment:
should we have like a constant defined higher as like a `defaultVis`?

Reply:
Thanks, I think it makes sense if the `defaultVis` is used in multiple places, since now it's only used once here, I will keep it this way